main_test.go

```diff
@@ -18,16 +18,15 @@ var dbsetup = false
 
 func TestMain(m *testing.M) {
 
-	// viper.Set("DbConnection", os.Getenv("TEST_DATABASE_URL"))
-	viper.Set("DbConnection", os.Getenv("dbname=ts"))
+	viper.Set("DbConnection", os.Getenv("TEST_DATABASE_URL"))
+	// viper.Set("DbConnection", os.Getenv("dbname=ts"))
```

Comment:
Just calling out that this could be a breaking change for other test environments.